dkarlovi/symfony-flex-docker/0.1/Makefile

```diff
@@ -0,0 +1,35 @@
+docker-start:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.$(APP_ENV).yaml up -d
+.PHONY: docker-up
+
+docker-stop:
```

Comment:
people should know that runing docker-stop will delete networks and volumes too. This is not harmless.